tools/build_cros_gcc

```diff
@@ -133,6 +145,10 @@
  ./$MPC.tar.gz 
 
  cd $GCC
+ for patch in isl-0.20; do
+   wget https://github.com/JL2210/patches/raw/master/gcc-8.2.0-$patch.patch
+   patch -p1 < gcc-8.2.0-$patch.patch
+ done
```

Comment:
Simpler?
```
wget https://github.com/JL2210/patches/raw/master/gcc-8.2.0-$ISL.patch
patch -p1 < gcc-8.2.0-$ISL.patch
```

Reply:
Not really. If I ever create more GCC patches, I could use those.

Reply:
Then do it the _"right"_ way:
```
PATCHES="$GCC-$ISL.patch"
for PATCH in $PATCHES; do
  wget https://github.com/JL2210/patches/raw/master/$PATCH
  patch -p1 < $PATCH
done
```

Reply:
OK.